api/tests/routes/native/v1/account_test.py

```diff
@@ -850,8 +858,7 @@ def send_two_requests(self, client, error_code):
             },
         )
 
-        assert response.status_code == 400
-        assert response.json["code"] == error_code
+        return response
```

Comment:
<!-- Issues represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 issue:** Oubli ? On veut assert des choses ici